
The usual gcc >= 3.4.x cought type mismatch problems.

  - Rene Rebe <rene@exactcode.de>

--- ./fpu/reg_ld_str.c.vanilla	2004-11-15 23:13:58.934640526 +0100
+++ ./fpu/reg_ld_str.c	2004-11-15 23:14:03.933843147 +0100
@@ -315,7 +315,7 @@
 
 
 /* Get a short from user memory */
-int  BX_CPP_AttrRegparmN(1)
+int  BX_CPP_AttrRegparmN(2)
 FPU_load_int16(bx_address _s, FPU_REG *loaded_data)
 {
   s16 s, negative;
